index.html

```diff
@@ -214,6 +214,26 @@ <h2 class="title-about">About<br>Myself</h2>
       </svg></div> 
       <hr width="100%" color="#DFE1E6" size="2px">     
   </section>
+  <footer class="contact-form">
+    <div class="first-container" >
+         <h2 class="contact-head">Contact me </h2>
+        <p class="para-contact">If you have an application you <br> are interested in developing, a <br>  feature that you need built or a <br> project that needs coding. <br> I’d love to help with it </p>
+    </div>
+    <form class="contact-list" action="https://formspree.io/f/xdobrbok" method="post">
+             <input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Enter Your name"  required>
+            <input type="email" id="email" name="user_email" placeholder="Enter your email" required>
+            <textarea name="message" id="msg" cols="24" rows="6" maxlength = "500" placeholder="Write your message here" required></textarea>
+            <button type="submit" class="btn">Get It Touch</button>
+        <img class="contact-img"  src="img/Ellipse 3.png" alt="cont-img" >
+        <img class="contact-img2"  src="img/Rectangle 4.png" alt="cont-img" >
+        <img class="contact-img3"  src="img/Ellipse 12.png" alt="cont-img" >
+        <img class="contact-img4"  src="img/Rectangle 7.png" alt="cont-img" >
+        <img class="contact-img5"  src="img/Ellipse 5.png" alt="cont-img" >
+        <img class="shape" src="img/Shape.png" alt="shape-img">
+    </form>
```

Comment:
- [ ] Everything else looks awesome apart from one small bit.Kindly note that there is a spacing around your entire footer and form section which is present in your design but is not present in the Figma design. Kindly ensure that you remove the margin/padding that is around your design to match the provided Figma design.

**This is how I am seeing your section right now:**

![image](https://user-images.githubusercontent.com/86318284/192729920-41447d65-3559-4bed-a847-a775e7091079.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/192730033-a0c82598-db49-4a27-a15d-4fb048efe216.png)